src/libstd/lib.rs

```diff
@@ -85,8 +85,9 @@
 //! # Contributing changes to the documentation
 //!
 //! Check out the rust contribution guidelines [here](
-//! https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md).
-//! The source for this documentation can be found on [Github](https://github.com/rust-lang).
+//! https://rustc-dev-guide.rust-lang.org/getting-started.html).
+//! The source for this documentation can be found on
+//! [Github](https://github.com/rust-lang/rustc-dev-guide).
```

Comment:
I think the "this documentation" is referring to the standard library, not the guide. I would use the link https://github.com/rust-lang/rust/.  Also, the `h` in GitHub should be capitalized.

Reply:
Ah, my bad! Will update it.